server/spec/models/grid_service_spec.rb

```diff
@@ -305,4 +305,25 @@
       expect(subject.affinity).to eq(['label!=type=ssd'])
     end
   end
+
+  describe '#deploy_outstanding?' do
```

Comment:
Also needs a spec for `returns false when deployments have finished`, i.e. both `started_at` and `finished_at`.